tile.go

```diff
@@ -8,10 +8,10 @@ import (
 )
 
 const (
-	DefaultEpislon    = 10.0
-	DefaultExtent     = 4096
-	DefaultTileBuffer = 64
-	MaxZ              = 20
+	DefaultEpislon            = 10.0
+	DefaultExtent             = 4096
+	DefaultTileBuffer float64 = 64
```

Comment:
What's the reason for adding the float64 here?
`Consts` generally don't have a type associated with them.

Reply:
I did it to be explicit since tile.Buffer expects a float64 but this const will be inferred as an int64 unless we change it to 64.0, which I'm also fine with.

Reply:
Let's change it to 64.0. I think that is better.

Reply:
done